### PR TITLE
Bayer2RGB network added to repository for debayerization.

### PR DESCRIPTION
**Pull request by asyatrhl**

### CODE REVIEW

**Review by rotx-maxim — CHANGES_REQUESTED** [review]

File: scripts/train_bayer2rgb_imagenet.sh
[the remark this answers is not in the document]
Comment (rotx-maxim) [reply scripts/train_bayer2rgb_imagenet.sh:1]: remove --data /data_ssd

File: scripts/evaluate_imagenet_bayer2rgb.sh
Comment (rotx-maxim) [inline scripts/evaluate_imagenet_bayer2rgb.sh:1]: Remove --data /data_ssd

**Review by rotx-maxim — CHANGES_REQUESTED** [review]

File: train_all_models.sh
Comment (rotx-maxim) [inline train_all_models.sh:1]: I think the final new line is missing from this file

**Review by rotx-maxim — APPROVED** [review]

**Review by MaximGorkem — APPROVED** [review]
Looks good.

**Review by ermanok — APPROVED** [review]
Looks good!

**Pull request merged**